### DISCUSSION TITLE
Incorrect documentation for removing search node

### DISCUSSION THREAD

**Discussion by trocade** [Security-Onion-Solutions/securityonion#3960] [Ideas] [answered]
I noticed there is a issue with the documentation on how remove a searchnode or heavynode from elasticsearch. I am not able to create a PR as requested so I will add it here and hope that is ok.

The documentation says to run the following command:
```
PUT _cluster/settings
{
  "persistent": {
    "search": {
      "remote": {
        "nodename": {
          "skip_unavailable": null,
          "seeds":null
        }
      }
    }
  }
}
```
But this will actually only reset the settings and it will still be queried by the search on kibana and timeout, to properly remove it the following should be done:
```
PUT _cluster/settings
{
  "persistent": {
    "cluster": {
      "remote": {
        "nodename": { 
          "mode": null,
          "seeds": null,
          "skip_unavailable": null,
          "transport": {
            "compress": null
          }
        }
      }
    }
  }
}
```

**Comment by dougburks** [comment, marked as answer]
Updated:
https://github.com/Security-Onion-Solutions/securityonion-docs/commit/18a2263c488a06f8f3aec66a4ed88b9bcdd44b3e#diff-38062bb6a2edcebe7b8344ce81119327e17383f0373058d4cc7277745b435348
